### ISSUE TITLE
[Migrate] :  Bootstrap to TailwindCSS

### ISSUE THREAD

**Issue by pkrawat1** [aviabird/angularspree#415]

**Comment by ankiitkumar07** [comment]
Hi @pkrawat1 
Let me know if I can work on it.

**Comment by pkrawat1** [comment]
Yes sure you can. I was kind of busy with some work and could not give much
time to it.

Pankaj Rawat Sent from Mobile

**Comment by pkrawat1** [comment]
There is already a Pr which can be picked #416 
@ankiitkumar07

**Comment by Rakesh-wagh** [comment]
@pkrawat1 
Let me know if I can work on it.

**Comment by pkrawat1** [comment]
Sure you can. You can refer this [PR](https://github.com/aviabird/angularspree/pull/416)
